samples/subsys/ipc/openamp_rsc_table/prj.conf

```diff
@@ -10,3 +10,7 @@ CONFIG_OPENAMP_MASTER=n
 
 CONFIG_SHELL=y
 CONFIG_SHELL_BACKEND_RPMSG=y
+CONFIG_SHELL_MINIMAL=y # Enabled to include Shell and Log for stm32mp157c_dk2 board
```

Comment:
1) This looks like a nice feature, although I am not very familiar with its usage. How does this coexist with the Shell already implemented? 

2) Regarding the sample itself
The sample seems already quite complex as it already integrates:
- Two TTY rpmsg devices (one for echo, one for the shell)
- one responder for the Linux client sample

Unless I missed something, this will create a third TTY rpmsg device on the Linux side. I wonder if it would be better to create a new sample for the feature instead of making this one overly complex. Notice that is not a request but a question for maintainers.

Additionally, I would expect to find information in the README file explaining how to run the sample.